### PR TITLE
Translation updates

### PR DESCRIPTION
**Pull request by jossim**
# Release Notes

Cherry picks spec changes from https://github.com/Oregon-State/nucore-osu/pull/185. Also removes `_upcase` from translation keys.

There's an adjustment in the override file here that needs to be reflected in other schools that are using it

### CODE REVIEW

**Comment by giladshanan** [comment]
Looking good, just a few spec failures

File: config/locales/en.yml [hunk 1]
```diff
@@ -29,17 +29,17 @@ en:
   facility_downcase: facility
   facilities_downcase: facilities
 
-  sso_id_upcase: NetID
-  sso_id_downcase: NetID
-  name_or_sso_id: name, NetID, or username
+  Sso_id: OID
+  sso_id_downcase: oid
+  name_or_sso_id: name, OID, or username
```
Comment (giladshanan) [inline config/locales/en.yml:34]:
```suggestion
  Sso_id: NetID
  sso_id_downcase: NetID
  name_or_sso_id: name, NetID, or username
```

File: config/locales/override/en.yml [hunk 2]
```diff
@@ -12,17 +12,18 @@ en:
...
+  #       one: Statement
+  #       other: Statements
+  #     facility_account:
+  #       one: Recharge Chart String
+  #       other: Recharge Chart Strings
+  #     nufs_account:
+  #       one: Chart String
+  #       other: Chart Strings
 
   # attributes:
```
Comment (giladshanan) [inline config/locales/override/en.yml:28]: attributes belongs nested under activerecord I believe

File: spec/system/admin/moving_transactions_spec.rb [hunk 3]
```diff
@@ -47,19 +50,19 @@
       find("input[value='#{od.id}']").click
     end
 
-    click_button "Reassign Chart Strings"
+    click_button "Reassign #{chart_string_name_upcase}"
```
Comment (giladshanan) [inline spec/system/admin/moving_transactions_spec.rb:53]: Should this be:
```suggestion
    click_button "Reassign #{chart_strings_name_upcase}"
```

**Review by giladshanan — CHANGES_REQUESTED** [review]
A few small changes, looks good over all

File: config/locales/override/en.yml (hunk 2 above)
Comment (jossim) [reply config/locales/override/en.yml]: Indeed! Done

**Review by giladshanan — APPROVED** [review]

**Pull request merged**